automation/taskcluster/decisionlib.py

```diff
@@ -223,8 +223,7 @@ def create(self):
         )
 
         task_id = taskcluster.slugId().decode("utf8")
-        SHARED.schedule_task(task_id, queue_payload)
-        return task_id
+        return task_id, queue_payload
```

Comment:
Isn't this change going to break the `master`/`release` pushes?

Reply:
Good call! I guess it was for debugging purpose. Let's change it back to `return task_id`

Reply:
The removal of `SHARED.schedule_task(task_id, queue_payload)` is ok?

Reply:
You're right, there's a second bug there. I didn't see it because I assumed `taskgraph` took care of digesting the tasks decisionlib created and then scheduled them. That's a strategy we applied in Fenix, for instance: https://github.com/mozilla-mobile/fenix/pull/5257. 

I took a closer look and I saw taskgraph is only called on PRs, and decisionlib otherwise. Thus, we do need to keep `SHARED.schedule_task(task_id, queue_payload)` around.

Thank you for spotting this bug! 👍 

That makes me wonder. What are the changes above about? (the ones in `__init__()`) Do we want to revert them too?

Reply:
Hey @eoger!

Just following up on this issue. Is there anything I can help with? (I'd like to make sure I'm not blocking anything 🙂)

Reply:
Sorry I forgot to reply to the earlier comment: yes I think we can revert all the changes in this file!

Reply:
Sorry about this, I forgot it here from during debugging, reverting all the changes to the file.